qa/064.out

```diff
@@ -0,0 +1,898 @@
+QA output created by 064
+
+=== pcp-pidstat with -f formatting of the timestamps
+Linux  4.4.0-31-generic  (ram-Lenovo)  08/05/16  x86_64    (4 CPU)
+Timestamp	UID	PID	usr	system	guest	%CPU	CPU	Command
+Tue Jun 13 14:34:30 2017	0	1	?	?	?	?	0	systemd
```

Comment:
Unfortunately, these timestamps are going to be incorrect until we fix the time.time() usage, but huge shoutout for adding qa!